spec/concerns/scopes_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'sort_n_params/concerns/scopes'
+
+RSpec.describe SortNParams::Scopes do
+  let(:test_class) do
+    CustomTable = Struct.new(:custom_field) do
+      include SortNParams::Scopes
+    end
+  end
+
+  context 'when you pass only an ordering array' do
+    subject { test_class.parse_array(["test_field","asc"]) }
+
+    it "builds the array correctly with the table_name param passed" do
```

Comment:
We may have to change this to:

`it "builds the array correctly without the table_name param passed" do`